### DISCUSSION TITLE
Update for compatibility with Crewai

### DISCUSSION THREAD

**Discussion by VMARGUET** [assafelovic/gpt-researcher#584] [General]
Hi there,
I'd love to run the gpt-researcher as a tool within a crewai agents project. i tried to set-it up but the dependencies of both projects are too different for them to be adapted easily.
Any update planned?
Thanks

**Comment by BradKML** [comment]
OpenHands or something more cutting edge on SWE-Bench would be good too https://www.swebench.com/
